web_teleop/launch/standalone.launch

```diff
@@ -1,7 +1,4 @@
 <launch>
-  <arg name="simulation" default="true"/>
-  <!-- For Simulation -->
-  <include file="$(find uw_gazebo)/launch/house_simulation.launch" if="$(arg simulation)"/>
 
   <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch"/>
```

Comment:
The `rosbridge_server` is already launched in [`rws_fetch.launch`](https://github.com/hcrlab/rws/blob/92ef76bbaa9b96345c1ea916fab8601be65a2470/launch/rws_fetch.launch#L8).

Reply:
This launch file is meant for running the interface completely standalone, without rws

Reply:
I see. Then which video server do you want to use? `web_video_server` or `ros_web_video`? I'm using `web_video_server` on port 8080 here and it is launched in `rws_fetch.launch`. We need to edit the file on Fetch if we choose to use `ros_web_video` instead.

Reply:
`web_video_server` makes sense (I was just looking in the wrong launch file and didn't see that you'd updated Fetch's rws launch to use it). I'll change the port back.